### PR TITLE
refactoring dev/prod env

### PR DESCRIPTION
**Pull request by fabiocicerchia**

### CODE REVIEW

**Comment by JellyBellyDev** [comment]
@fabiocicerchia are you really asking me for the review or is it still in draft?

**Comment by fabiocicerchia** [comment]
@JellyBellyDev yes, we can review this :)

File: composer.json [hunk 1]
```diff
@@ -64,7 +64,8 @@
         "allow-plugins": {
             "phpstan/extension-installer": true,
             "symfony/flex": true,
-            "symfony/runtime": true
+            "symfony/runtime": true,
+            "ocramius/package-versions": true
```
Comment (garak) [inline composer.json:68]: This is unsorted, it should be in the first place

**Review by garak — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -2,28 +2,40 @@ version: "3.8"
...
+    build:
+      context: .
+      dockerfile: sys/docker/alpine-nginx/Dockerfile
+      network: host
     container_name: badge-poser-nginx
     depends_on:
       - phpfpm
+    env_file:
+      - ./.env
+      - ./.env.local
```
Comment (drAlberT) [inline docker-compose.yml:14]: Are these used by nginx too?

File: docker-compose.yml [hunk 3]
```diff
@@ -2,28 +2,40 @@ version: "3.8"
...
+    build:
+      context: .
+      dockerfile: sys/docker/alpine-phpfpm/Dockerfile
+      network: host
     container_name: badge-poser-phpfpm
     depends_on:
       - redis
+    env_file:
+      - ./.env
+      - ./.env.local
```
Comment (JellyBellyDev) [inline docker-compose.yml:34]: `.env.test` to run tests in local?

File: sys/docker/alpine-phpfpm/Dockerfile.dev [hunk 4]
```diff
@@ -1,8 +1,8 @@
-FROM php:8.0-fpm-alpine
+FROM php:8.0.16-fpm-alpine
```
Comment (JellyBellyDev) [inline sys/docker/alpine-phpfpm/Dockerfile.dev:1]: we can delete this file, no?

**Review by JellyBellyDev — CHANGES_REQUESTED** [review]

**Comment by JellyBellyDev** [comment]
I tried to test `make start` and i have an error with:
`docker build -t awsbuilder -f sys/docker/common/Dockerfile.aws-ecs-ci .`
```
 => [2/3] RUN apt-get update &&      apt-get install -y ca-certificates curl &&     rm -rf /var/lib/apt/lists/*                                                                                                                                                            6.8s
 => ERROR [3/3] RUN curl -O https://s3.amazonaws.com/amazoncloudwatch-agent/debian/amd64/latest/amazon-cloudwatch-agent.deb &&     dpkg -i -E amazon-cloudwatch-agent.deb &&     rm -rf /tmp/* &&     rm -rf /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agen  52.1s
------
 > [3/3] RUN curl -O https://s3.amazonaws.com/amazoncloudwatch-agent/debian/amd64/latest/amazon-cloudwatch-agent.deb &&     dpkg -i -E amazon-cloudwatch-agent.deb &&     rm -rf /tmp/* &&     rm -rf /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-config-wizard &&     rm -rf /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl &&     rm -rf /opt/aws/amazon-cloudwatch-agent/bin/config-downloader:
#6 0.137   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
#6 0.137                                  Dload  Upload   Total   Spent    Left  Speed
100 57.5M  100 57.5M    0     0  1138k      0  0:00:51  0:00:51 --:--:-- 1884k
#6 51.99 dpkg: error processing archive amazon-cloudwatch-agent.deb (--install):
#6 51.99  package architecture (amd64) does not match system (arm64)
#6 52.01 Errors were encountered while processing:
#6 52.01  amazon-cloudwatch-agent.deb
------
executor failed running [/bin/sh -c curl -O https://s3.amazonaws.com/amazoncloudwatch-agent/debian/amd64/latest/amazon-cloudwatch-agent.deb &&     dpkg -i -E amazon-cloudwatch-agent.deb &&     rm -rf /tmp/* &&     rm -rf /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-config-wizard &&     rm -rf /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl &&     rm -rf /opt/aws/amazon-cloudwatch-agent/bin/config-downloader]: exit code: 1
make: *** [.docker_img_deps] Error 1
```

**Comment by fabiocicerchia** [comment]
@JellyBellyDev the problem with `make start` should be fixed. It was due to arm64 arch, I guess you're on M1, right?

**Comment by JellyBellyDev** [comment]
ok now the build works, but if I try to modify the sources I don't notice the modification on the web side.

**Pull request merged**